butterknife/src/test/java/butterknife/OnTouchTest.java

```diff
@@ -33,6 +34,7 @@
         + "  private Test target;\n"
         + "  private View view1;\n"
         + "  @UiThread\n"
+        + "  @SuppressLint(\"ClickableViewAccessibility\")\n"
```

Comment:
If you want I can move this annotation directly to the `onTouch(..)` implementation to reduce the scope.